src/CSV/CSVWriter.php

```diff
@@ -28,11 +29,9 @@ class CSVWriter
      */
     protected $expect_schema;
 
-    /**
-     * @var array
-     */
-    protected $options = [
-        'write_utf8_bom' => FALSE
+    protected array $options = [
```

Comment:
Technically, can't add a strict typehint to a `protected` property without a BC break - if someone is extending the class (e.g. with their own default / custom options) they would have to add the typehint too.